README.md

```diff
@@ -557,6 +557,14 @@ $job = $client->jobs()->show($jobId);
 ```
 Returns the job.
 
+##### Wait for a job to finish
```

Comment:
Please explain in more detail how it "waits". Make it clear that this polls for the job being finished with sleeps in between, so people don't have incorrect expectations of how this behaves.

Reply:
Added to the README